eclipse-collections/src/main/java/org/eclipse/collections/impl/collector/Collectors2.java

```diff
@@ -672,6 +1218,19 @@ private Collectors2()
                 EMPTY_CHARACTERISTICS);
     }
 
+    /**
+     * <p>Returns all elements of the stream that return true when evaluating the predicate with the parameter.
+     * The results are return in the Collection created by evaluating the Supplier.</p>
```

Comment:
I made all of the methods consistent as you suggested using the second variation above.